platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/Property.java

```diff
@@ -22,9 +24,14 @@
     @Retention(RetentionPolicy.SOURCE)
     public @interface VISIBILITY {}
 
-    //line-cap
+    /**
+     * `line-cap`: The display of line endings.
+     */
+    //A cap with a squared-off end which is drawn to the exact endpoint of the line.
```

Comment:
Same thing here. I would invert this, start the header with `//` and the individual comments with `/** */`